pathd/path_pcep_pcc.c

```diff
@@ -868,8 +891,8 @@ void specialize_outgoing_path(struct pcc_state *pcc_state, struct path *path)
 	was_created = path->update_origin == SRTE_ORIGIN_PCEP;
 
 	path->pcc_id = pcc_state->id;
-	path->go_active = is_delegated;
-	path->is_delegated = is_delegated;
+	path->go_active = (is_delegated & pcc_state->is_best);
```

Comment:
Should be `is_delegated && pcc_state->is_best` with `&` and without parenthesis.

Reply:
done